app/cafe.py

```diff
@@ -0,0 +1,20 @@
+from datetime import date
+
+from app.errors import NotVaccinatedError
+from app.errors import NotWearingMaskError
+from app.errors import OutdatedVaccineError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError

```

Comment:
The `NotVaccinatedError` should be raised with a descriptive message. According to the checklist, use descriptive error messages when raising exceptions. Consider modifying the exception initialization in `errors.py` to include a message, or pass a message here.